development/image-syncer/README.md

```diff
@@ -0,0 +1,63 @@
+# Image Syncer
+
+## Overview
+
+Image Syncer is used to **safely** copy container images from one registry to another. 
+The main use-case is to preserve images from third party registries in our own registry that we can relay on.
+
+Syncing process steps:
+- pull image from source
+- check if image exists in target
+- if it does not exist - re-tag image and push to target registry
+- if it exists compare IDs - if they are different synchronisation is interrupted, 
+it means that something is wrong - source image was changed (that should not happen)
+
+These steps guarantee that images in our registry are immutable.
+
+## Usage
+
+To run it, use:
+```bash
+go run main.go \ 
+    --images-file={path to a yaml file containing sync definition} \
+    --target-key-file={path to a json key file} \
+    --dry-run=true
+```
+
+***Defintion file***
+
+Image syncer as an input parameter takes a file containing following structure: 
```

Comment:
```suggestion
image-syncer as an input parameter takes a file having the following structure: 
```

Does the file **have** the structure below or does it **contain** it? (Is it the whole file or is it just a part of it?)